packages/next-swc/crates/napi/src/mdx.rs

```diff
@@ -0,0 +1,43 @@
+use mdxjs::{compile, Options};
+use napi::bindgen_prelude::*;
+
+pub struct MdxCompileTask {
+    pub input: String,
+    pub option: Buffer,
```

Comment:
Maybe `options` (with `s`? 🤔🤷‍♂️)

Reply:
it won't be a huge matter since this is internal context anyway.